dotCMS/src/integration-test/java/com/dotmarketing/portlets/contentlet/transform/ContentletTransformerTest.java

```diff
@@ -235,8 +261,41 @@ public void Test_Constant_fields_are_in_map() throws Exception {
         
     }
 
-    
-    
+
+    @Test
+    public void Transformer_Backwards_Compatibility_Test()
+            throws DotDataException, DotSecurityException {
+        //final Contentlet con = APILocator.getContentletAPI()
+          //      .findContentletByIdentifier("2cc0ad0f-d5e9-4c4c-a243-227dd2c49ba3", false, 1L,
+            //            APILocator.systemUser(), false);
+        List<Contentlet> list = APILocator.getContentletAPI().findAllContent(250, 650);
+
+        list = list.stream().filter(Objects::nonNull).collect(Collectors.toList());
+        assertFalse("I was expecting at least 20 contentlets returned from the index",list.isEmpty());
+        final List<Map<String, Object>> transformedList = new ContentletToMapTransformer(list).toMaps();
+        final List<Map<String, Object>> transformedList2 = new DotContentletTransformer(list).toMaps();
+
+        assertEquals(transformedList.size(), transformedList2.size());
+        for(int i=0; i < list.size(); i++){
+            final Contentlet original = list.get(i);
+            final Map<String,Object> originalTransformation = transformedList.get(i);
+            final Map<String,Object> transformation2= transformedList2.get(i);
+
+            for (String key1 : originalTransformation.keySet()) {
+                final Object object1 = originalTransformation.get(key1);
+                final Object object2 = transformation2.get(key1);
+
+                if (object1.equals(ContentletCache.CACHED_METADATA) || object1.equals(ContentletCache.EMPTY_METADATA)) {
+                   System.out.println(original.getStringProperty("metaData"));
+                    System.out.println(original.getContentType());
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [System.out.println is used](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5596402)